lerna.json

```diff
@@ -4,7 +4,8 @@
     "motion",
     "screen-reader",
     "storage",
-    "text-zoom"
+    "text-zoom",
+    "browser"
```

Comment:
I think we are putting them in alphabetical order, not by adding order, should be on top